docs/src/pages/_tags.md

```diff
@@ -0,0 +1,10 @@
+Mongolid supports **ActiveRecord** pattern.
+
+[![Build Status](https://travis-ci.org/leroy-merlin-br/mongolid.svg?branch=master)](https://travis-ci.org/leroy-merlin-br/mongolid)
+[![Coverage Status](https://coveralls.io/repos/github/leroy-merlin-br/mongolid/badge.svg?branch=master)](https://coveralls.io/github/leroy-merlin-br/mongolid?branch=master)
+[![Latest Stable Version](https://poser.pugx.org/zizaco/mongolid/v/stable.png)](https://packagist.org/packages/zizaco/mongolid)
+[![Total Downloads](https://poser.pugx.org/zizaco/mongolid/downloads.png)](https://packagist.org/packages/zizaco/mongolid)
+[![Latest Unstable Version](https://poser.pugx.org/zizaco/mongolid/v/unstable.png)](https://packagist.org/packages/zizaco/mongolid)
+[![License](https://poser.pugx.org/zizaco/mongolid/license.png)](https://packagist.org/packages/zizaco/mongolid)
+
+[//]: # ([![SensioLabsInsight]&#40;https://insight.sensiolabs.com/projects/25636a94-9a5d-4438-bd5e-9f9694104529/small.png&#41;]&#40;https://insight.sensiolabs.com/projects/25636a94-9a5d-4438-bd5e-9f9694104529&#41;)
```

Comment:
This is commented because the link is broken.
So I don't know if still make sense keep this badge.